src/builder.ts

```diff
@@ -1148,16 +1148,23 @@ export class SchemaBuilder {
       if (typeof val === "string") {
         return this.addDynamicScalar(methodName, val, block);
       }
-      // @ts-ignore
-      block[methodName] = (...args: any[]) => {
-        const config = isList ? [args[0], { list: isList, ...args[1] }] : args;
```

Comment:
I need to review this existing `args` logic more and see what's being lost in this PR currently. Pretty sure something.